hail/src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -1,5 +1,6 @@
 package is.hail.expr.ir
 
+import com.google.api.gax.rpc.InvalidArgumentException
```

Comment:
delete this?

Reply:
Oh, that was a mistake where I typed "Invalid" instead of "Illegal", must have autoimported.

Reply:
yeah, I do that all the time!